### PR TITLE
DOCSP-4379 Validate and clarify intro in README

### PR DESCRIPTION
**Pull request by cbush**
- Ran through the steps in README to validate their current accuracy
- Some improvements/clarifications made

Staged:
https://docs-mongodbcom-staging.corp.mongodb.com/stitch/bush/docsp-4379-readme/sdk/swift/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ target '<Your Target Name>' do
     # optional: for using the AWS service
     pod 'StitchSDK/StitchAWSService', '= 5.1.0'
     # optional: for using the Firebase Cloud Messaging service
-    pod 'StitchSDK/StitchFCMService', '= 5.1.0'
+    pod 'StitchFCMService', '= 5.1.0'
```
Comment (adamchel) [inline README.md]: [nit] I don't think this should have changed, is this actually how it works now?

File: README.md [hunk 2]
```diff
@@ -69,26 +69,33 @@ Open the `.xcworkspace` file generated by `pod install` to access your project w
...
+3. Go to your app in Stitch via Atlas by clicking 'Stitch' in the left side pane and clicking your app.
+3. Copy your Stitch App ID from the top of the left side pane.
+4. Enable anonymous authentication by going to the Users page in the left side pane, selecting the Providers tab, and enabling the "Allow users to log in anonymously" option.
 
-#### Set up a project in XCode/CocoaPods using Stitch
+#### Set up a project in Xcode/CocoaPods using Stitch
+
+1. Download and install [Xcode](https://developer.apple.com/xcode/). 
+
+- Note: Xcode 10.1 (10B61) is currently the maximum supported version. To install an old version of Xcode, see [https://developer.apple.com/download/more/](https://developer.apple.com/download/more/).
```
Comment (adamchel) [inline README.md]: @jsflax when you do the release, make sure you update this

File: README.md [hunk 3]
```diff
@@ -69,26 +69,33 @@ Open the `.xcworkspace` file generated by `pod install` to access your project w
...
-5. Run `pod install`
-6. Open the generated `.xcworkspace` file
-7. Your app project will have all the necessary dependencies configured to communicate with MongoDB Stitch.
-8. To use basic Stitch features, `import StitchCore` in a source file.
-9. To access a remote MongoDB instance via Stitch, `import StitchRemoteMongoDBService` in a source file.
+See above for the list of optional Stitch service pods that you may add to your Podfile.
+
+5. Run `pod install`.
+6. Open the generated `.xcworkspace` file. Your app project will have all the necessary dependencies configured to communicate with MongoDB Stitch.
+7. To use basic Stitch features, `import StitchCore` in a source file.
```
Comment (adamchel) [inline README.md]: Might also be worth mentioning "To create BSON documents and BSON values like ObjectId, `import MongoSwift` in a source file"

**Comment by cbush** [comment]
Linting passed, evg otherwise broken until Xcode 10.2 installed. Merging.

**Pull request merged**